config/src/main/java/com/typesafe/config/impl/AbstractConfigNode.java

```diff
@@ -0,0 +1,15 @@
+package com.typesafe.config.impl;
+
+import com.typesafe.config.ConfigNode;
+
+abstract class AbstractConfigNode implements ConfigNode {
+    final private Token token;
```

Comment:
then `render` would of course render the whole list of tokens. but maybe we have no multi-token nodes?